tests/test_converters/test_calendars/test_hijri/test_hijri_parser.py

```
@@ -0,0 +1,76 @@
+    "Foo 383",
+    # wrong format
+    "2024-10-02",
+    # year month day not supported
+    "901 Rabīʿ I",
+    "901 Rabīʿ I 12",
+]
+
+
+@pytest.mark.parametrize("date_string", error_cases)
+def test_should_error(date_string):
+    with pytest.raises(Exception):
```

Ruff (0.8.0)

`pytest.raises(Exception)` should be considered evil

(B017)